native/packages/react-native-bpk-component-phone-input/src/__snapshots__/BpkDialingCodeAccessoryView-test.ios.js.snap

```diff
@@ -0,0 +1,247 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`iOS BpkDialingCodeAccessoryView should render correctly 1`] = `
+<View
+  accessibilityComponentType="button"
+  accessibilityLabel={undefined}
+  accessibilityTraits={
+    Array [
+      "button",
+    ]
+  }
+  accessible={true}
+  hitSlop={undefined}
+  nativeID={undefined}
+  onLayout={undefined}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+  style={
+    Object {
+      "justifyContent": "flex-end",
+    }
+  }
+  testID={undefined}
+>
+  <View
```

Comment:
One thing I have noticed with `TouchableNativeFeeback` (android) vs `BpkTouchableOverlay` (bpk specific ios) is that the android component clone the view you provide and applies props whereas the ios component wraps. Do we think this is a problem?

Reply:
Not sure, but `TouchableOpacity` from `React Native` is doing the wrapping right?

Reply:
The one thing that stops us from cloning is that we need to add another view along side the content. Given that it's iOS (in general more powerful devices) it might not be a big deal.